extensions/amp-accordion/1.0/amp-accordion.js

```diff
@@ -242,7 +255,7 @@ const bindHeaderShimToElement = (element) => HeaderShim.bind(null, element);
  */
 function ContentShimWithRef(
   sectionElement,
-  {hidden, id, role, 'aria-labelledby': ariaLabelledBy},
+  {id, role, 'aria-labelledby': ariaLabelledBy, 'aria-hidden': ariaHidden},
```

Comment:
Do we need to remove `aria-hidden` inclusions from this file?

Reply:
Thanks, yep, I did another pass which removes from the amp-side, thanks!